pombola/south_africa/templates/core/person_detail.html

```diff
@@ -7,297 +7,236 @@
-          {% empty %}
-            <li>Not a member of any party</li>
-          {% endfor %}
-        {% else %}
-          <li>Not a member of any party</li>
-        {% endif %}
-      {% endfor %}
-    </ul>
-
-    {% regroup object.constituencies|dictsort:"kind" by kind as constituency_groups %}
-    {% for entry in constituency_groups %}
-    <h3>{{ entry.grouper }}</h3>
-    <ul>
-        {% for area in entry.list %}
-            <li><a href="{{ area.get_absolute_url }}">{{ area.name }}</a></li>
-        {% endfor %}
-    </ul>
-    {% endfor %}
 
-    {% if email_contacts %}
-      <h3>Email</h3>
-      <ul>
-        {% for email in email_contacts %}
-          <li class="word-break-all">{{ email|urlizetrunc:50 }}</li>
-        {% endfor %}
-      </ul>
-    {% endif %}
-
-    {% if address_contacts %}
-      <h3>Address</h3>
-      <ul>
-        {% for address in address_contacts %}
-          <li>{{ address }}</li>
-        {% endfor %}
-      </ul>
-      {% endif %}
-
-    {% if phone_contacts %}
-      <h3>Phone</h3>
-      <ul>
-        {% for phone in phone_contacts %}
-          <li><a href="tel:{{ phone }}">{{ phone }}</a></li>
-        {% endfor %}
-      </ul>
-    {% endif %}
-
-    {% if fax_contacts %}
-      <h3>Fax</h3>
-      <ul>
-        {% for fax in fax_contacts %}
-        <li>{{ fax }}</li>
-      {% endfor %}
-      </ul>
-    {% endif %}
-
-    {% if twitter_contacts %}
-      <h3>Twitter</h3>
-      <ul>
-        {% for twitter in twitter_contacts %}
-          <li><a href="http://twitter.com/{{ twitter }}">@{{ twitter }}</a> on Twitter</li>
-        {% endfor %}
-      </ul>
-    {% endif %}
-
-    {% if object.constituency_offices %}
-      <h3>Constituency Offices</h3>
-      <ul>
-        {% for office in object.constituency_offices %}
-          <li><a href="{{ office.get_absolute_url }}">{{ office.name }}</a></li>
-        {% empty %}
-          <li>Not associated with any constituency offices</li>
-        {% endfor %}
-      </ul>
-    {% endif %}
-
-  </div>
-</div>
 {% endblock %}
 
 {% block subcontent %}
-<div class="tabs">
-  <ul class="tab-links">
-    <li><a class="active" href="#profile">Profile</a></li>
-    {% with past_positions=person.position_set.all.political.currently_inactive %}
-    {% if past_positions %}
-    <li><a href="#experience">Experience</a></li>
-    {% endif %}
-    {% if hansard.count or question.count or committee.count %}
-    <li><a href="#appearances">Appearances</a></li>
-    {% endif %}
-    {% if object.interests_register_entries.exists %}
-    <li><a href="#membersinterests">Register of Interests</a></li>
-    {% endif %}
-  </ul>
-
-  <div id="profile" class="tab-content tab-active">
-
-    <div class="person-summary">
-      {% if person.date_of_death %}
-        <p>Died {{ person.date_of_death }}</p>
-      {% endif %}
-      {{ person.summary }}
-    </div>
+</div> <!-- .large-container -->
```

Comment:
These are tiny points, but worth mentioning because it took me a while to figure out: I think it might be worth having a {% comment %} block here just to explain why the enclosing `div`s need to be closed and reopened, since it's a bit surprising to see.  I'd also make sure that the indentation is such that the block looks like:

```
{% block subcontent %}
    </div> <!-- .large-container -->
  </div> <!-- page-wrapper -->
  ...
  <div class="page-wrapper wrapper">
    <div class="large-container">
    ...
{% endblock %}
```